scripts/generate_panelite_content.py

```diff
@@ -0,0 +1,21 @@
+"""
+Helper script to convert and copy example notebooks into JupyterLite build.
+"""
+import glob
+import pathlib
+
+import nbformat
+
+nbs = glob.glob('./examples/**/*/*.ipynb') + glob.glob('./examples/*/*.ipynb')
+install = nbformat.v4.new_code_cell(source="import piplite\nawait piplite.install(['panel'])")
+del install['id']
```

Comment:
I think `id` will be mandatory for Notebook in the future. I have at least seen some warnings indicating it.

Reply:
We strip out `id` in all notebooks currently so for the time being we do the same here.